webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/BootstrapChecks.scala

```diff
@@ -86,22 +98,25 @@ class SequentialImmediateBootStrapChecks(_checkActions: BootstrapChecks*) extend
     .appendSeparator(" ")
     .appendMillis()
     .appendSuffix(" ms")
-    .toFormatter();
+    .toFormatter()
 
   @throws(classOf[UnavailableException])
-  override def checks(): Unit = checkActions.zipWithIndex.foreach {
-    case (check, i) =>
-      val start = System.currentTimeMillis
-      val msg   = if (BootstrapLogger.logEffect.isDebugEnabled) {
-        s"[#${i}] ${check.description}"
-      } else {
-        s"${check.description}"
-      }
-      BootstrapLogger.logEffect.info(msg)
-      check.checks()
-      BootstrapLogger.logEffect.debug(
-        msg + s": OK in [${formatter.print(new Duration(System.currentTimeMillis - start).toPeriod)}] ms"
-      )
+  override def checks(): Unit = {
```

Comment:
I still think this should be a ZIO

Reply:
it could, but that has nothing to do with the bug here. It's a perfect cleaning for next minor version